lib/kitchen/driver/oci/blockstorage.rb

```diff
@@ -141,6 +150,18 @@ def volume_details(volume)
           )
         end
 
+        def volume_clone_details(volume, clone_volume_name)
+          OCI::Core::Models::CreateVolumeDetails.new(
+            compartment_id: oci.compartment,
+            availability_domain: config[:availability_domain],
+            display_name: clone_volume_name,
+            defined_tags: config[:defined_tags],
+            size_in_gbs: volume[:size_in_gbs],
```

Comment:
to my surprise, if you specify a larger size than that of the volume being cloned, it just works.  it appears to clone the data and drop it in a larger volume.  the opposite is not true if you specify a smaller size.  in that case, you get 

```
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Failed to complete #create action: [Requested new volume size 50 GB must be greater than or equal to existing volume size 100 GB] on default-ol86
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

This results in needing to destroy/re-create the instance because the volume processing is essentially a post-create task.  To me, there are two options: leave this as-is, or remove `size_in_gbs` and force users into keeping the size the same.

I think the "as-is" approach is fine for now.  I don't want to outright recommend removing functionality to prevent this error.  That feels like overkill to me.  Maybe make a note in the README on the `size_in_gbs` parameter indicating this requirement when specified with the `volume_id` parameter.

Reply:
Sounds good, I just updated the README to note on `size_in_gbs` to warn the user about that.

Reply:
Not annoying at all.  I appreciate you taking the time to make the requested changes.